backend/app/tests/integration/base_test_class.py

```diff
@@ -118,3 +118,17 @@ def login_test_user(self):
         )
         data = json.loads(response.data)
         self.user_access_token = data["payload"]["access_token"]
+
+    def create_comment(self):
```

Comment:
why here in the base_test_class? Isn't your base test class supposed to be as generic as possible?

- [ ] Extract this and add it to comment_test.py